### PR TITLE
Swerve tweaks

### PR DESCRIPTION
**Pull request by jonahsnider**

### CODE REVIEW

File: src/main/java/frc/robot/swerve/SwerveModule.java [hunk 1]
```diff
@@ -19,6 +19,7 @@
 import frc.robot.misc.util.sensors.SensorUnitConverter;
 
 public class SwerveModule {
+  private static final double MAX_VELOCITY = 4.5;
```
Comment (saikiranra) [inline src/main/java/frc/robot/swerve/SwerveModule.java]: I believe we have the SDS MK4 L1 modules, which would run at 4.1148 Meters per second.

https://www.swervedrivespecialties.com/products/mk4-swerve-module

File: src/main/java/frc/robot/swerve/SwerveModule.java [hunk 2]
```diff
@@ -61,7 +62,10 @@ public void setDesiredState(SwerveModuleState state) {
 
     state = CtreModuleState.optimize(state, steerMotorPosition);
 
-    final var rotations = Units.radiansToRotations(state.angle.getRadians());
+    double rotations =
+        (Math.abs(state.speedMetersPerSecond) <= MAX_VELOCITY * 0.01)
+            ? steerMotorPosition.getRadians()
```
Comment (saikiranra) [inline src/main/java/frc/robot/swerve/SwerveModule.java]: I would switch this to this.previousRotations instead of indexing this off the current angle.

Comment (saikiranra) [reply src/main/java/frc/robot/swerve/SwerveModule.java]: The logic here is we want the last valid setting for the angle to carry forward, even if the current wheel angle changes at low speeds

**Review by saikiranra — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/swerve/SwerveModule.java (hunk 1 above)
Comment (jonahsnider) [reply src/main/java/frc/robot/swerve/SwerveModule.java]: this number should also be converted to feet/sec since we're not using SI anymore

**Pull request merged**